Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,24 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const result = tasks
+    .map(item => item.duration / 60)
+    .filter(item => item >= 2)
+    .reduce((totalRate, item) => totalRate + item * hourlyRate, 0);
   console.log(tasks, hourlyRate);
+  console.log(result);
+  return result;
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
-
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+const roundedEarnings = earnings
+  .toFixed(2)
+  .toString()
+  .replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1.');
```

Comment:
What is this line doing?

Reply:
This line is formatting the number as currency. To do that I wrote "Currency Formating in javascript" and found this website : https://blog.abelotech.com/posts/number-currency-formatting-javascript/
But i replaced '$1,' with '$1.' But this maybe make the code confusing.

Reply:
Well it is not needed because line 63 already rounds it to 2 decimals and line 66 already puts a € before it.